client/src/test/java/org/apache/rocketmq/client/impl/consumer/ConsumeMessageConcurrentlyServiceTest.java

```diff
@@ -62,9 +65,7 @@
 import static org.mockito.ArgumentMatchers.anyLong;
 import static org.mockito.ArgumentMatchers.anyString;
 import static org.mockito.ArgumentMatchers.nullable;
-import static org.mockito.Mockito.doReturn;
-import static org.mockito.Mockito.spy;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```

Comment:
Aggregating all imports is not a good practice.

Reply:
Thanks for reminding, it may be automatically added by IDE, I will correct it later.

Reply:
I have corrected it, best regards.